controllers/admin.js

```diff
@@ -0,0 +1,14 @@
+
+// import usersInfo from '../models/usersInfo';
+// import mentorsInfo from '../models/usersInfo';
```

Comment:
@cynmumbua , please remove all commented lines of code,. they are redundant and are not useful